tests/Fixer/Alias/NoMixedEchoPrintFixerTest.php

```diff
@@ -282,8 +285,11 @@ public function testDefaultConfig()
      */
     public function testWrongConfig($wrongConfig)
     {
-        $this->expectException('PhpCsFixer\ConfigurationException\InvalidFixerConfigurationException');
-        $this->expectExceptionMessageRegExp('#^\[no_mixed_echo_print\] Expected array of element \"use\" with value \"echo\" or \"print\", got \"array \([\S\s]*\)\".$#');
+        $this->setExpectedExceptionRegExp(
```

Comment:
`->setExpectedException*` method do exist in bot phpunit versions, even if they are deprecated.
`->expectException*` methods do not exist on lower phpunit version used by lowers PHP engines, so you cannot use it. as simple as that.